requirements.txt

```
@@ -1,7 +1,8 @@
 jsonschema==2.3.0
 validators==0.14.2
 filetype==1.0.5
-insightconnect-integrations-plugin-spec-tooling~=1.0
+insightconnect-integrations-plugin-spec-tooling~=1.3
 requests==2.22.0
 urlextract==0.14.0
-typing==3.7.4.1
\ No newline at end of file
+typing==3.7.4.1
+PyYAML~=5.3.1
```

```suggestion
PyYAML~=5.3.0
```
I believe this is the correct notation for all patch versions